controllers.controllers.js

```diff
@@ -0,0 +1,47 @@
+const { getTopics } = require("./models.models");
+
+exports.getAllTopics = (req, res, next) => {
+  getTopics()
+    .then((topics) => {
+      res.status(200).send({ topics });
+    })
+    .catch((error) => {
+      next(error);
+    });
+};
+
+exports.getAllEndpoints = (req, res, next) => {
+  const endpoints = {
```

Comment:
Here you want to be making a separate json file, requiring it in and then sending it back, rather than writing all this code here.